site/labs/conditionals/conditionals.rst

```diff
@@ -2,53 +2,90 @@
 Conditionals
 ************
 
+* Feel free to use your laptop if you have it
```

Comment:
This gets repeated a lot. Does Sphinx have any way to inject a template, or another file verbatim? Updating this will be a pain.

Reply:
I'll check, but if not, meh.